### PR TITLE
Fixes for Pickup Creation

### PR DESCRIPTION
**Pull request by pepperkick**

### CODE REVIEW

File: src/server/services/pickup-queue/statuses/ready-up/create-pickup.js [hunk 1]
```diff
@@ -103,7 +104,15 @@ export default async function createPickup(props) {
         status: 'waiting',
         classes: mapValues(
           pickupQueue.classes,
-          classPlayers => classPlayers.filter(player => !players.includes(player.id)),
+          classPlayers => classPlayers.filter((player) => {
```
Comment (HenriBeck) [inline src/server/services/pickup-queue/statuses/ready-up/create-pickup.js]: Why is this change needed?

Comment (pepperkick) [reply src/server/services/pickup-queue/statuses/ready-up/create-pickup.js]: I was getting "players.include" is not a function errors everytime a pickup was created.

Comment (HenriBeck) [reply src/server/services/pickup-queue/statuses/ready-up/create-pickup.js]: Totally forgot that players is an object with the classes inside
